src/index.html

```diff
@@ -5,10 +5,50 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
Although not mandatory, it's recommended to include crossorigin attribute with the value of 'anonymous' for the preconnect link to fonts.gstatic.com. This can help with CORS issues.